tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -850,6 +854,86 @@ def test_run_call_pipeline_service_create_with_failure_policy(
             gca_pipeline_state.PipelineState.PIPELINE_STATE_SUCCEEDED
         )
 
+    @pytest.mark.parametrize(
+        "job_spec",
+        [_TEST_PIPELINE_SPEC_JSON, _TEST_PIPELINE_SPEC_YAML, _TEST_PIPELINE_JOB],
+    )
+    @pytest.mark.parametrize("sync", [True, False])
+    def test_run_call_pipeline_service_create_with_input_artifact(
```

Comment:
I wonder if it is necessary to add this test? An alternative option is to add `input_artifacts` in the existing `test_run_call_pipeline_service_create`.

Reply:
Done.